build.gradle.kts

```diff
@@ -23,17 +23,18 @@ repositories {
 dependencies {
 	implementation("mysql:mysql-connector-java:8.0.33")
 	implementation("org.springframework.boot:spring-boot-starter-data-jpa")
-	implementation("org.springdoc:springdoc-openapi-webmvc-core:1.8.0")
+	implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.0.2")
```

Comment:
Se não for utilizar a a dependencia considere retirar ela do projeto

Reply:
Acredito que agora com a nova solução começou a utilizar